UK/vATIS/UK - Scottish AC.json

```diff
@@ -1,107 +1,132 @@
 {
-  "Name": "UK - Scottish AC",
-  "Composites": [
+  "name": "UK - Scottish AC",
+  "id": "b1431b6d-4010-48e4-bd25-3e0ae3a1d5fc",

```

Comment:
The id isn't required as far as I know. Not a big issue but increases the file size.